tap_github/schemas/issue_events.json

```diff
@@ -954,10 +954,166 @@
+                "null",
+                "string"
+              ]
+            },
+            "login": {
+              "type": ["null", "string"]
+            },
+            "id": {
+              "type": ["null", "integer"]
+            },
+            "node_id": {
+              "type": ["null", "string"]
+            },
+            "avatar_url": {
+              "type": ["null", "string"]
+            },
+            "gravatar_id": {
+              "type": [
+                "null",
+                "string"
+              ]
+            },
+            "url": {
+              "type": ["null", "string"]
+            },
+            "html_url": {
+              "type": ["null", "string"]
+            },
+            "followers_url": {
+              "type": ["null", "string"]
+            },
+            "following_url": {
+              "type": ["null", "string"]
+            },
+            "gists_url": {
+              "type": ["null", "string"]
+            },
+            "starred_url": {
+              "type": ["null", "string"]
+            },
+            "subscriptions_url": {
+              "type": ["null", "string"]
+            },
+            "organizations_url": {
+              "type": ["null", "string"]
+            },
+            "repos_url": {
+              "type": ["null", "string"]
+            },
+            "events_url": {
+              "type": ["null", "string"]
+            },
+            "received_events_url": {
+              "type": ["null", "string"]
+            },
+            "type": {
+              "type": ["null", "string"]
+            },
+            "site_admin": {
+              "type": ["null", "boolean"]
+            },
+            "starred_at": {
+              "type": ["null", "string"]
+            }
+          }
+        },
+        "name": {
+          "type": ["null", "string"]
+        },
+        "description": {
+          "type": [
+            "null",
+            "string"
+          ]
+        },
+        "external_url": {
+          "type": ["null", "string"]
+        },
+        "html_url": {
+          "type": ["null", "string"]
+        },
+        "created_at": {
+          "type": ["null", "string"],
+          "format": "date-time"
+        },
+        "updated_at": {
+          "type": ["null", "string"],
+          "format": "date-time"
+        },
+        "permissions": {
+          "type": ["null", "object"],
+          "properties": {
+            "issues": {
+              "type": ["null", "string"]
+            },
+            "checks": {
+              "type": ["null", "string"]
+            },
+            "metadata": {
+              "type": ["null", "string"]
+            },
+            "contents": {
+              "type": ["null", "string"]
+            },
+            "deployments": {
+              "type": ["null", "string"]
+            }
+          },
+          "additionalProperties": {
+            "type": ["null", "string"]
+          }
```

Comment:
The `additionalProperties` should be either `true` or `false`.

Reply:
Removed as unused